Tools/YamlCreate.ps1

```diff
@@ -358,6 +359,7 @@ Function Get-InstallerFile {
 
     # Create a new web client for downloading the file
     $_WebClient = [System.Net.WebClient]::new()
+    $_WebClient.Headers.Add("User-Agent", "Microsoft-Delivery-Optimization/10.1")
```

Comment:
Just a note: the wininet downloader now uses the format from https://github.com/microsoft/winget-cli/pull/1839 (or it will in v1.3).

I don't think you should change it here, just wanted to make sure everyone heard.